src/test/resources/emailconfig.properties

```diff
@@ -0,0 +1,12 @@
+#Gmail Login Credentials
+mailAddress= YOUR_EMAIL_ADDRESS
```

Comment:
How does it get injected here? Need some notes on need to update this file.

Reply:
I think we need to add this in readme file?

Reply:
Lets add at least at the class level in javadoc mentioning the need to update the config file with parameters that are mandatory

Reply:
We may even catch the condition where the email, pwd are still default - then throw an exception RuntimeException("emailconfig not initialized with credentials!")